msaBase/utils/constants.py

```diff
@@ -1,8 +1,10 @@
+import os
+
 SERVICE_TOPIC = "service-config"
 REGISTRY_TOPIC = "registry-config"
 PROGRESS_TOPIC = "progress"
 DATABASE_UPDATE_TOPIC = "database-update"
-PUBSUB_NAME = "redispubsub"
+PUBSUB_NAME = os.environ["DAPR_PUBSUB_COMPONENT_NAME"]
```

Comment:
In this case we need to add DAPR_PUBSUB_COMPONENT_NAME to the env in each service. It is better to use some default parameter